### PR TITLE
Removing deprecated test configs from tests

### PR DESCRIPTION
**Pull request by ferigis**
Removing deprecated test configs from tests

### CODE REVIEW

File: test/ejabberd_tests/tests/presence_SUITE.erl [hunk 1]
```diff
@@ -611,15 +611,15 @@ remove_roster(Config, UserSpec) ->
     end.
 
 set_versioning(Versioning, VersionStore, Config) ->
-    Host = escalus_ct:get_config(ejabberd_domain),
+    Host = ct:get_config({hosts, mim, domain}),
     RosterVersioning = escalus_ejabberd:rpc(gen_mod, get_module_opt, [Host, mod_roster, versioning, false]),
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/presence_SUITE.erl]: According to **Elvis**:
> Line 615 is too long:     RosterVersioning = escalus_ejabberd:rpc(gen_mod, get_module_opt, [Host, mod_roster, versioning, false]),.

File: test/ejabberd_tests/tests/metrics_api_SUITE.erl [hunk 2]
```diff
@@ -310,4 +311,3 @@ assert_counter_inc(_Name, Inc, Counter1, Counter2) when Counter1 + Inc =:= Count
 
 get_diffs(L1,L2) ->
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/metrics_api_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 312

File: test/ejabberd_tests/tests/vcard_SUITE.erl [hunk 3]
```diff
@@ -1085,5 +1085,3 @@ vcard_result_mapping(_) -> undefined.
 get_utf8_city() ->
     %% This is the UTF-8 of Москва
     <<208,156,208,190,209,129,208,186,208,178,208,176>>.
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl [hunk 4]
```diff
@@ -212,7 +212,7 @@ request_search_fields(Config) ->
     escalus:story(
       Config, [{alice, 1}],
       fun(Client) ->
-              Domain = escalus_config:get_ct(ejabberd_domain),
+              Domain = ct:get_config({hosts, mim, domain}),
               DirJID = <<"vjud.",Domain/binary>>,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 216

File: test/ejabberd_tests/tests/vcard_SUITE.erl (hunk 3 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl [hunk 5]
```diff
@@ -269,7 +269,7 @@ search_wildcard(Config) ->
     escalus:story(
       Config, [{bob, 1}],
       fun(Client) ->
-              Domain = escalus_config:get_ct(ejabberd_domain),
+              Domain = ct:get_config({hosts, mim, domain}),
               DirJID = <<"vjud.",Domain/binary>>,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 273

File: test/ejabberd_tests/tests/vcard_SUITE.erl (hunk 3 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl [hunk 6]
```diff
@@ -143,7 +143,7 @@ user_doesnt_exist(Config) ->
     escalus:story(
       Config, [{alice, 1}],
       fun(Client) ->
-              Domain = escalus_config:get_ct(ejabberd_domain),
+              Domain = ct:get_config({hosts, mim, domain}),
               BadJID = <<"nonexistent@",Domain/binary>>,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 147

File: test/ejabberd_tests/tests/ejabberdctl_SUITE.erl [hunk 7]
```diff
@@ -131,7 +131,9 @@ init_per_group(vcard, Config) ->
     end;
 
 init_per_group(roster_advanced, Config) ->
-    case escalus_ejabberd:rpc(gen_mod,get_module_opt,[ct:get_config(ejabberd_domain), mod_roster, backend, mnesia]) of
+    case escalus_ejabberd:rpc(gen_mod,get_module_opt,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/ejabberdctl_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 134

File: test/ejabberd_tests/tests/vcard_SUITE.erl (hunk 3 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

File: test/ejabberd_tests/tests/ejabberdctl_SUITE.erl [hunk 8]
```diff
@@ -122,7 +122,7 @@ end_per_suite(Config) ->
 
 init_per_group(vcard, Config) ->
     case escalus_ejabberd:rpc(gen_mod,get_module_opt,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/ejabberdctl_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 124

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl [hunk 9]
```diff
@@ -250,7 +250,7 @@ search_some(Config) ->
     escalus:story(
       Config, [{bob, 1}],
       fun(Client) ->
-              Domain = escalus_config:get_ct(ejabberd_domain),
+              Domain = ct:get_config({hosts, mim, domain}),
               DirJID = <<"vjud.",Domain/binary>>,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 254

File: test/ejabberd_tests/tests/presence_SUITE.erl [hunk 10]
```diff
@@ -611,15 +611,15 @@ remove_roster(Config, UserSpec) ->
...
     RosterVersionOnDb = escalus_ejabberd:rpc(gen_mod, get_module_opt, [Host, mod_roster, store_current_id, false]),
     escalus_ejabberd:rpc(gen_mod, set_module_opt, [Host, mod_roster, versioning, Versioning]),
     escalus_ejabberd:rpc(gen_mod, set_module_opt, [Host, mod_roster, store_current_id, VersionStore]),
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/presence_SUITE.erl]: According to **Elvis**:
> Line 618 is too long:     escalus_ejabberd:rpc(gen_mod, set_module_opt, [Host, mod_roster, store_current_id, VersionStore]),.

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl [hunk 11]
```diff
@@ -419,7 +419,9 @@ search_result_item_tuples(Stanza) ->
     _ItemTups = item_tuples(ReportedFieldTups, XChildren).
 
 is_vcard_ldap()->
-    ldap==escalus_ejabberd:rpc(gen_mod,get_module_opt,[ct:get_config(ejabberd_domain), mod_vcard, backend, mnesia]).
+    ldap==escalus_ejabberd:rpc(gen_mod,get_module_opt,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 422

File: test/ejabberd_tests/tests/vcard_SUITE.erl [hunk 12]
```diff
@@ -462,7 +462,7 @@ search_open_limited(Config) ->
     escalus:story(
       Config, [{alice, 1}],
       fun(Client) ->
-              Server = escalus_ct:get_config(ejabberd_domain),
+              Server = ct:get_config({hosts, mim, domain}),
               DirJID = <<"directory.",Server/binary>>,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 466

File: test/ejabberd_tests/tests/vcard_SUITE.erl (hunk 3 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

File: test/ejabberd_tests/tests/shared_roster_SUITE.erl [hunk 13]
```diff
@@ -154,7 +156,7 @@ stop_roster_module(_)->
     ok.
 
 get_auth_method() ->
-    XMPPDomain = ct:get_config(ejabberd_domain),
+    XMPPDomain = ct:get_config({hosts, mim, domain}),
     escalus_ejabberd:rpc(ejabberd_config, get_local_option,[{auth_method, XMPPDomain}]).
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/shared_roster_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 160

File: test/ejabberd_tests/tests/vcard_SUITE.erl (hunk 3 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 1087

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl (hunk 5 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 273

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl [hunk 14]
```diff
@@ -235,7 +235,7 @@ search_empty(Config) ->
     escalus:story(
       Config, [{alice, 1}],
       fun(Client) ->
-              Domain = escalus_config:get_ct(ejabberd_domain),
+              Domain = ct:get_config({hosts, mim, domain}),
               DirJID = <<"vjud.",Domain/binary>>,
```
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 239

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl (hunk 5 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 273

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl (hunk 14 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 239

Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 239

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl (hunk 5 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 273

Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 273

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl (hunk 14 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 239

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl (hunk 5 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 273

File: test/ejabberd_tests/tests/vcard_simple_SUITE.erl (hunk 14 above)
Comment (gadgetci) [inline test/ejabberd_tests/tests/vcard_simple_SUITE.erl]: According to **Elvis**:
> Missing space right "," on line 239

File: test/ejabberd_tests/test.config [hunk 15]
```diff
@@ -7,14 +7,8 @@
...
 {ejabberd_node, 'mongooseim@localhost'}.
-{ejabberd2_node, 'ejabberd2@localhost'}.
 {ejabberd_cookie, ejabberd}.
-%% the main XMPP domain served by the tested ejabberd/MongooseIM
-{ejabberd_domain, <<"localhost">>}.
-{ejabberd_secondary_domain, <<"localhost.bis">>}.
-{ejabberd_reloaded_domain, <<"sogndal">>}.
-{ejabberd_metrics_rest_port, 5288}.
-{ejabberd2_metrics_rest_port, 5289}.
+%% the main XMPP domain served by the tested ejabberd/MongooseIM.
```
Comment (michalwski) [inline test/ejabberd_tests/test.config]: Let's remove the comment as it is no longer relevant

**Pull request merged**